code/modules/reagents/reagent_containers/glass.dm

```diff
@@ -169,45 +176,50 @@
 	icon_state = "beaker"
 	item_state = "beaker"
 	g_amt = 500
-	w_type = RECYK_GLASS

```

Comment:
RECYK_GLASS var was moved to the reagent_containers/glass parent for obvious reasons